### PR TITLE
#164076491 Fix permissions for some views

### PR DESCRIPTION
**Pull request by MuhweziDeo**
What does this PR do?
implement fixes for permissions to allow unauthenticated users to be able to access some endpoints

#### Description of Task to be completed?
- change IsAuthenticated permission to IsAuthenticatedOrReadOnly for comment, comment reply, user profile list, following, followers views

#### How should this be manually tested?
- Fetch and checkout into this branch with `git fetch origin bg-fix-permissions-164076491 && git checkout bg-fix-permissions-164076491`
- run `python manage.py run server`
- perform a get request on the endpoint like `localhost:8000/api/articles/articleslug/comments` without passing in a token

#### What are the relevant pivotal tracker stories?
[##164076491](https://www.pivotaltracker.com/story/show/#164076491)

### CODE REVIEW

File: authors/apps/comments/commentReplyViews.py [hunk 1]
```diff
@@ -1,12 +1,14 @@
...
+from rest_framework.permissions import ( IsAuthenticated,
+  IsAuthenticatedOrReadOnly )
 from rest_framework.response import Response
-from .serializers import ReplyLikeSerializer, ReplyCommentSerializer, ReplyHistorySerailizer
+from .serializers import (ReplyLikeSerializer, ReplyCommentSerializer,
+    ReplyHistorySerailizer )
 from authors.apps.profiles.models import Profile
 from authors.apps.articles.models import Article
 from authors.apps.authentication.models import User
-from .models import CommentReplyLike, Comment, CommentLike, CommentReply
```
Comment (calvinpete) [inline authors/apps/comments/commentReplyViews.py:9]: @MuhweziDeo Would you please clarify as to why you removed the CommentLike model import statement?

Comment (MuhweziDeo) [reply authors/apps/comments/commentReplyViews.py:9]: It was an unused import @calvinpete

**Review by calvinpete — APPROVED** [review]
LGTM

**Review by armstrongsouljah — APPROVED** [review]
LGTM!

**Review by kdaglas — APPROVED** [review]

**Pull request merged**